### PR TITLE
Ingame UI

### PR DESCRIPTION
**Pull request by GDWR**
Added basic UI to show current level (Currently is always 1) and a key count.

closing #22

### CODE REVIEW

File: ingame_ui.py [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+    """
+    Used to display in-game interface features like Key count and Current Level.
+    """
+
+    def __init__(self, player_inventory: PlayerInventory):
+        self.player_inv = player_inventory
+        self.inv_sprite = arcade.SpriteList()
+        self.key_sprite = arcade.Sprite("assets/sprites/key.png", 2)
+        self.inv_sprite.append(self.key_sprite)
+        self.colour = int(0x22), int(0x3D), int(0x28)
```
Comment (SavagePastaMan) [inline ingame_ui.py:17]: don't need to use `int` here, they're already integers

**Review by SavagePastaMan — CHANGES_REQUESTED** [review]
other than the one comment, lgtm

File: ingame_ui.py (hunk 1 above)
Comment (GDWR) [reply ingame_ui.py:17]: True, I was being lazy in converting Hex to Ints. the middle is 3D and it's also not the same base.

Comment (GDWR) [reply ingame_ui.py:17]: Do you want this changing? or is my lazy hex conversion fine.

Comment (SavagePastaMan) [reply ingame_ui.py:17]: wdym, they're ints, just represented as hex. just remove the `int` call and it should work the exact same

Comment (GDWR) [reply ingame_ui.py:17]: Oh shit, you rite. Dunno why I thought they would be a Hex obj

**Review by SavagePastaMan — APPROVED** [review]
lgtm

**Pull request merged**